blog/arquillian-cube-1.15.3.textile

```diff
@@ -0,0 +1,43 @@
+---
+date: 2018-03-29
+author: dipak-pawar
+---
+
+h3. Highlights of this release
+
+p. Apart from several bug fixes, this release contains three awesome features.
+
+h3. Fabric8 Maven Plugin integration
+
+p. From now you can use "fabric8-maven-plugin":https://maven.fabric8.io/ to build images and generate k8s/openshift artifacts and let cube use it in your tests.
+
+However you have to configure @fabric8-maven-plugin@ using @arquillian.xml@ as per your need.
+
+Here's an example:
+
+bc(prettify).. <extension qualifier="openshift">
+    <property name="cube.fmp.build">true</property>
+</extension>
+
+p. By enabling this property, cube will build images, generate resources using @fabric8-maven-plugin@ when running test from @Maven@ or @IDE@.
+
+p(info). %User can configure to run @fabric8-maven-plugin@ only from @IDE@ (not from maven build) by setting property @cube.fmp.build.disable.for.mvn@ to @true@. Look "here":http://arquillian.org/arquillian-cube/#_fabric8_maven_plugin_integration_with_cube for more configuration options.%
```

Comment:
Is property `true` by default? Maybe it should be true?

Reply:
no there is no point to make it true. we need to embedded maven build only if user is using fmp.

Reply:
So by default it runs from IDE and maven build. Ok.

Reply:
notice that there might be a situation when a user doesn't have fmp in his pom.xml. In this case, we should not execute embedded maven build from cube.